packages/expo-random/src/Random.ts

```diff
@@ -20,12 +20,20 @@ function assertByteCount(value: any, methodName: string): void {
 /**
  * Generates completely random bytes using native implementations. The `byteCount` property
  * is a `number` indicating the number of bytes to generate in the form of a `Uint8Array`.
+ * Falls back to `Math.random` during development to prevent issues with React Native Debugger.
  * @param byteCount - A number within the range from `0` to `1024`. Anything else will throw a `TypeError`.
  * @return An array of random bytes with the same length as the `byteCount`.
  */
 export function getRandomBytes(byteCount: number): Uint8Array {
   assertByteCount(byteCount, 'getRandomBytes');
   const validByteCount = Math.floor(byteCount);
+  if (__DEV__) {
```

Comment:
Perhaps try detecting if remote JS debugging is enabled instead?

Reply:
I wasn't able to find a legit way to detect if the app is being run with remote debugging enabled. Do you have any suggestions?

Reply:
What about `if (global.__REMOTEDEV__)`?

Reply:
I saw that on a SO page but couldn't find it in code or documentation anywhere.

Reply:
https://twitter.com/sebastienlorber/status/1420782232118120453?s=20

Reply:
> just be sure to check for web (if applicable)

I wasn't sure how to tell if this is applicable since I'm not working on the web in my project.

Reply:
`Platform.OS !== 'web'`